Assignment-07-Product Designer/styles/css/assignment7.css

```diff
@@ -0,0 +1,150 @@
+body {
+  background-color: black;
+  margin: 0;
+  padding: 0;
+}
+
+.initial_header {
+  margin-top: 40px;
+  display: flex;
+  flex-direction: row-reverse;
+  color: #ffff;
+  margin-bottom: -8px;
+  font-family: "Roboto", "sans-serif";
+  font-weight: 600;
+  width: 88.4%;
+}
+
+.flex_container_shrutti {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #ffff;
+  margin-top: 60px;
+}
+
+.shrutti {
+  font-size: 60px;
+  font-weight: 700;
+  font-family: "Roboto", "sans-serif";
+}
+
+.shrutti_first_name {
+  font-size: 60px;
+  font-weight: 300;
+  font-family: "Roboto", "sans-serif";
+}
+
+.designer {
+  font-size: 25px;
+  color: #ffff;
+  font-family: "Roboto", "sans-serif";
+  margin-top: 20px;
+}
+
+.product {
+  color: rgba(74, 144, 226, 255);
+}
+
+.flex_container_items {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(0, 0, 0);
+  margin-top: 50px;
+}
+
+.first_block {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 20px;
+}
+
+.second_block {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.image_content {
+  width: 25%;
+  height: 100%;
+  margin-left: 20px;
+  margin-bottom: 20px;
+}
+
+.icons {
+  width: 5%;
+  height: 26px;
+}
+
+.show_off_items {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin-top: 50px;
+  /* border: 2px solid #ffff; */
+  width: 87%;
+  margin: auto;
+}
+
+.show_off {
+  font-size: 200%;
+  color: rgba(206, 206, 206, 255);
+  text-align: center;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  margin-bottom: 60px;
+}
+
+.show_off_name {
+  color: rgba(184, 233, 134, 255);
+}
+
+.flex_container_footer {
+  display: flex;
+  flex-direction: column;
+  margin-top: 50px;
+}
+
+.separator {
+  color: white;
```

Comment:
Uniform color code properties
and UX to UI match

Reply:
uniform color code updated to hex color code